metadata/db.go

```diff
@@ -44,6 +44,11 @@ const (
 	// version which are additions and compatible with
 	// prior version of the same schema.
 	dbVersion = 3
+
+	// Policy* represent content sharing policy
```

Comment:
I think we can just keep this as a boolean. The configuration can remain a string, but we should keep the internal use as simple as possible. We can have the variable be `shared`, so internally it can default to false when not initialized, but the config can default to shared.

Reply:
Agree and done, though I keep const of  Policy* to hold those policy strings for logging

Reply:
These strings can be next to the configuration, nothing needs to be added to the metadata public interface for this other than the option

Reply:
If more mode is planned to be added (e.g. "shared-across-namespaces-with-particular-label"), maybe we can use string even for the internal struct.